index.html

```diff
@@ -46,32 +47,34 @@
             </li>
           </ul>
         </div>
+        <!-- did mistake ... fix -->
+        <button class="donate_now">Donate Now</button>
+      
+      </header>
 
-        <button>Donate Now</button>
-
-    </div>
 
     <div class="content">
-        <div class="main">
+        <main class="main">
           <div class="alert">
             You've helped us give <span class="text-highlight">72</span> bikes
             to refugees and asylum seekers so far. <strong>Thank you!</strong>
           </div>
 
           <div class="hero">
+            <!-- still looking how to add background bike photo , nevermind founded :)-->
             <h1>Bikes for Refugees</h1>
             <p>
               Providing donated bikes and accessories to refugees and asylum
-              seekers in Scotland.
+              seekers in England.
             </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
+            <button class="donate_now">Donate a bike today</button>
+            <button class="volunteer">Volunteer</button>
           </div>
 
           <div>
             <h2 class="heading-underline">Learn more</h2>
-
-            <div>
+           <!-- created parent element gave class to this elemet -->
+            <div class="flexbox">
```

Comment:
It's usually better to pick a class name for css classes that describes the content your user will see. E.g. here you could say `class=articles` instead of `class=flexbox`.